### DISCUSSION TITLE
在iOS的Safari浏览器中点击容器端口无法跳转

### DISCUSSION THREAD

**Discussion by 001ProMax** [1Panel-dev/1Panel#6725] [Q&A]
### 1Panel 版本

v1.10.18-lts

### 问题描述

在iOS的Safari浏览器中点击容器端口无法跳转

### 重现步骤

1、使用iOS的Safari浏览器打开1Panel
2、点击容器端口，无法跳转

### 期待的正确结果

在新标签页打开对应端口

**Comment by ssongliu** [comment]
感谢反馈，页面最底部的使用手册和项目地址能正常打开吗？我确认一下是不是被拦截了

**Comment by 001ProMax** [comment]
能打开

> 感谢反馈，页面最底部的使用手册和项目地址能正常打开吗？我确认一下是不是被拦截了

**Comment by 001ProMax** [comment]
> 感谢反馈，页面最底部的使用手册和项目地址能正常打开吗？我确认一下是不是被拦截了

macOS的Safari也能跳转，只有iOS上不行
